.pylintrc

```diff
@@ -0,0 +1,34 @@
+[DESIGN]
```

Comment:
I guess this complete file is here to better match the analysis criteria with our own coding styling right? I assume most of the values are defaults or slightly variations of them.

Reply:
yes this is in case we run pylint on the code. I have adapted some default values to match our criteria. My hope is that landscape will import that file and apply it to the pylint call they run

Reply:
problem is that landscape.io is sooo slow that I still have no check results.

I'm not sure if that service will survive. Thus I would be ok if we switch to some other service, e.g codacy.

We also should check how we integrate with github services vs. apps. Travis as an example will move us over to travis.ci.com, we just have to wait. I don't know if landscape and friends already supports github apps as they should.

Would be great if you have some experience here to share your thoughts 

Thanks

Reply:
To be honest I don't have experience or background to help here. However I can spend some time tomorrow to get some ;)  and experiment with landscape, some alternatives and do some digging.

Reply:
Thanks for the offer. If you are interested in code analytic it's surely nice to see what they do. These are all external services which needs approval by the github org maintainers. Thus playing with them is only easily possible in your personal github account. For the moment I'll stick with landscape.io and will jump to something more productive. Thanks